README.md

```diff
@@ -0,0 +1,62 @@
+# merkle-trie [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0)
+Merkle Trie used by [CodeChain](https://github.com/CodeChain-io/codechain) and [Foundry](https://github.com/CodeChain-io/foundry)
+
+## Usage
+
+The rust-merkle-trie crate provides merkle trie implementations. CodeChain and Foundry use a Merkle Trie to hold the states with authentication. We differ from the merkle patricia trie in Ethereum; details can be found [here](https://github.com/CodeChain-io/codechain/blob/master/spec/Merkle-Trie.md). 
```

Comment:
If we are going to use lower case for merkle trie, let's make it lower case in the beginning of the second sentence.